aip/general/0193.md

```diff
@@ -62,11 +62,90 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
+### Error messages
 
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+The [`google.rpc.Status`][Status] error payload **should** contain two
+error messages.
+
+1. The payload **must** contain an error message in the `message` field
+   found at the root of [`Status`][Status].
+1. The payload **should** contain
+   [`google.rpc.LocalizedMessage`][LocalizedMessage] in `details` on
+   `Status`. This is the location of the second error message.
+
+The error message on `Status` should,
+
+- Be developer-facing, human-readable, and in English
+- Both expalin the error and offer an actionable resolution to it
+  ([citation](https://cloud.google.com/apis/design/errors#error_model))
+
+Once launched the value of `message` at the root-level of `Status`
+**must not** change for the error payload to be backwards compatible.
```

Comment:
@jskeet points out that this statement may need further clarification, e.g. definition around what "backward compatible" means in this context.

Reply:
I would also say we need clarification around what the "error payload" constitutes e.g. the `message` is paired with the `code` OR the `message` is paired with the specific error mode (e.g. invalid location errors), so that services can reasonably keep track of what `message` an "error payload" should use.

Not a blocking comment for this PR.

Reply:
Good point. However, this statement was removed, which is not to say that we do not need to address this concern at a later date.